sys/linux/vmw_vmci_amd64.const

```diff
@@ -0,0 +1,4 @@
+# AUTOGENERATED FILE
```

Comment:
This file is now replaced by vmw_vmci.txt.const. Remove it.

Reply:
Done